providers/ses/src/lib/ses.provider.spec.ts

```diff
@@ -12,6 +12,7 @@ const mockConfig = {
 
 const mockNovuMessage = {
   to: ['test@test2.com'],
+  replyTo: 'test@test1.com',
```

Comment:
I think that this is not enough to test if `replyTo` is being configured in SES. 
In line 96 we should modify the spy to check that the proper params and values are passed by checking with the method `toHaveBeenCalledWith` and the values passed in the `mockNovuMessage`.
Unfortunately it wasn't done properly before but if not doing it we can't validate this change. 🙁